src/librustc_feature/active.rs

```diff
@@ -555,6 +555,9 @@ declare_features! (
     // Allows limiting the evaluation steps of const expressions
     (active, const_eval_limit, "1.43.0", Some(67217), None),
 
+    /// Allows the use of `#[target_feature]` on safe functions.
+    (active, target_feature_11, "1.43.0", Some(69098), None),
```

Comment:
Note to myself: change this to 1.44.0
```suggestion
    (active, target_feature_11, "1.44.0", Some(69098), None),
```